evalml/automl/automl_algorithm/iterative_algorithm.py

```diff
@@ -65,7 +65,7 @@ def next_batch(self):
 
         next_batch = []
         if self._batch_number == 0:
-            next_batch = [pipeline_class(parameters=self._transform_parameters(pipeline_class, {}), random_state=self.random_state)
+            next_batch = [pipeline_class(parameters=self._transform_parameters(pipeline_class, {}), random_seed=self.random_seed)
```

Comment:
`self.random_seed == self._random_state`. We get `self.random_seed` from the parent class. `self._random_state` is used to sample params in the first batch if the user provides pipelines with frozen hyperparams. I'll get rid of `self._random_state` since we don't need it anymore!